shinydashboard/lantern/ui.R

```diff
@@ -437,8 +499,20 @@ ui <- dashboardPage(
                       attributeFilter: [\"tabindex\"]
                     });
                 }
-              }
+              }            
             }
+            
+            let selectInputButtons = document.querySelectorAll(\"select.shiny-bound-input\")
+            for (let selectInput of selectInputButtons) {
+              selectInput.setAttribute('aria-label', 'Use the arrow keys to naviate the filter menu.')
+            }
+          }
+          
+          if (mutation.target.id === \"show_filters\") {
+            let dropDownButtons = document.getElementsByClassName(\"dropdown-toggle\")
+            for (let dropDownButton of dropDownButtons) {
+              dropDownButton.setAttribute('aria-label', 'Dropdown filter menu button. Press the down arrow key to open the filter menu, use the tab or arrow keys to navigate through options, press enter to select a filter option, and use the escape key to close the filter menu.')
```

Comment:
Is it possible to just have this read out when you're on the dropdown without it being open? I tried navigating with VoiceOver and it was kind of a lot to have this read every time you switched options for the dropdown. If this is the only way to do it then I think it's fine, but just wanted to ask and see if there was an alternative!